docs/responses/source.rst

```diff
@@ -6,100 +6,47 @@ Give us the source of the status.
 .. note::
     The API equivalent is :code:`status_source`.
 
+
 HTTP Code
 ^^^^^^^^^
 
 This source is returned when **all the following cases** are met:
 
-- We can't extract the expiration date from :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
+- We can't extract the expiration date from
+  :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
 - The :code:`INACTIVE` status is the one returned by other methods.
-- :func:`PyFunceble.lookup.http_code.HTTPCode.get` outputs is different from the default one
-  (:code:`XXX`) and the other methods gives the :code:`INACTIVE` status.
+- :func:`PyFunceble.lookup.http_code.HTTPCode.get` outputs is different from
+  the default one (:code:`XXX`) and the other methods gives the
+  :code:`INACTIVE` status.
+
 
 SYNTAX
 ^^^^^^
 
-This source is always returned when the domain has the status :code:`INVALID` or in the case that we are only checking for syntax instead of availability.
-The usage of this source comes from the comparison of the element against our domain, IP or URL syntax validation system.
+This source is always returned when the domain has the status :code:`INVALID`
+or in the case that we are only checking for syntax instead of availability.
+The usage of this source comes from the comparison of the element against our
+domain, IP or URL syntax validation system.
+
 
 DNSLOOKUP
 ^^^^^^^^^
 
-This source is always returned when the taken decision of the status of the domain/IP comes from :func:`PyFunceble.lookup.dns.DNSLookup.request` outputs.
-
-SPECIAL
-^^^^^^^
-
-As :code:`PyFunceble` grows, I thought that a bit of filtering for special cases would be great.
-So I introduced the SPECIAL source.
-
-
-.. note::
-    Please consider all 3 digits number that are listed in this section as the HTTP status code catched by :func:`PyFunceble.lookup.http_code.HTTPCode.get`.
-
-.. warning::
-    Do not want those rules ? You can use the following to disable them.
-
-    * :code:`-ns|--no-special` arguments from the CLI.
-    * :code:`no_special: True` into your local configuration file.
-
-:code:`*.000webhostapp.com`
-"""""""""""""""""""""""""""
-
-- All :code:`410` are returned as :code:`INACTIVE`.
-
-:code:`*.blogspot.*`
-""""""""""""""""""""
-
-- All :code:`404` are returned as :code:`INACTIVE`
-- All :code:`301` which are blocked by Google or does not exist are returned as :code:`INACTIVE`
-- All :code:`302` which are blocked by Google are returned as :code:`INACTIVE`
-
-:code:`*.canalblog.com`
-"""""""""""""""""""""""
+This source is always returned when the taken decision of the status of the
+domain/IP comes from :func:`PyFunceble.lookup.dns.DNSLookup.request` outputs.
```

Comment:
Not your fault but let's update it!

:class:`PyFunceble.query.dns.query_tool.DNSQueryTool` should be correct now 😄 

In other words (previous is weird): `func` is now `class` and `PyFunceble.lookup.dns.DNSLookup.request` is now `PyFunceble.query.dns.query_tool.DNSQueryTool`.

Reply:
Could you throw these into a [todo](https://github.com/spirillen/PyFunceble/issues)?

Reply:
Should have been solved 

Are you moving <https://github.com/funilrys/PyFunceble/projects/7#card-52531257>